docs/schema.md

```diff
@@ -460,11 +460,11 @@ databases, in the form of the `id` field. This allows one database to claim that
 its own entry describes the same vulnerability as one or more entries in other
 databases.
 
-Two vulnerabilities can be described as aliases if a potential patch that
-addresses one of the vulnerabilities (and no other vulnerabilities) will also
-address the other vulnerability, and vice versa. Aliases may be used for
-vulnerabilities affecting different packages or ecosystems as long as they
-follow this definition.
+Two vulnerabilities can be described as aliases if they affect any given
+software component the same way: either both vulnerabilities affect the software
+component or neither do. And, consequently, a potential patch that addresses one
```

Comment:
Grammar nit: don't start sentences with a conjunction.

```suggestion
component or neither do. Consequently, a potential patch that addresses one
```

Reply:
Or more invasively, 

"A subsequent patch addresses both of the vulnerabilities (and no others), and vice versa."

Reply:
I like that!

Reply:
Adjusted in f84a635